### ISSUE TITLE
Multiple maps hard to use. Impl relies heavily on React context.

### ISSUE THREAD

**Issue by rektide** [tomchentw/react-google-maps#21]
This code makes excessive use of an undocumented, unsupported feature in React: the context. It stores, for example, the reference to the Google Maps objects in the context.

This makes using multiple maps on the page extremely hard/impossible. If you have a Marker, which Map does the Marker get added to? Well, that's going to depend on which map is in the React context right now.

React context is in general a big no-no.

**Comment by tomchentw** [comment]
Just use two different root component to wrap the code:
```html
<div>
  <YourCustomComponentWithGoogleMapsMixin>
    <Map>
      <Marker/>
    <Map>
  <YourCustomComponentWithGoogleMapsMixin/>
  <YourCustomComponentWithGoogleMapsMixin>
    <Map>
      <Marker/>
    <Map>
  <YourCustomComponentWithGoogleMapsMixin/>
</div>
```

I agree context wouldn't be a good solution, just depending on how React goes in the future :)

**Comment by tomchentw** [comment]
Released [v1.0.0](https://github.com/tomchentw/react-google-maps/releases/tag/v1.0.0) and closed this now.
